packages/flutter_driver/lib/src/common/request_data.dart

```diff
@@ -26,7 +26,8 @@ class RequestData extends Command {
 
   @override
   Map<String, String> serialize() => super.serialize()..addAll(<String, String>{
-    'message': message,
+    if (message != null)
```

Comment:
This is the fix

Reply:
Ahhh, OK, I was misunderstanding the `!` because I wasn't seeing the != null around it; it's used to remove the `?` from the type of the variable. Cool beans!